copy-bundles.sh

```diff
@@ -0,0 +1,11 @@
+#/bin/sh
```

Comment:
is this .sh file a "script" to copy files from one folder to another? and also, is there a script that runs through each individuals repo to build their webpack prod config and copy it to your destination folder?

Reply:
yeah, it's just a script to copy the files.  I haven't written a script to build everyone's webpack prod yet, though...

Reply:
gotcha!